src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -53,9 +55,15 @@ def load_arguments(self, _):
 
     with self.argument_context('backup container show') as c:
         c.argument('name', container_name_type, options_list=['--name', '-n'], help='Name of the container. You can use the backup container list command to get the name of a container.')
+        c.argument('backup_management_type', options_list=['--backup-management-type'], help=backup_management_type_help)
```

Comment:
Actually, you can also define CLIArgumentType for backup_management_type like container_name_type and reuse it in your commands.